### ISSUE TITLE
#Loot monsters like take-out-only chests

### ISSUE THREAD

**Issue by NeroOneTrueKing** [Chris-plus-alphanumericgibberish/dNAO#771]
And by that, I mean don't put all their items under the broad heading "equipment".

**Comment by Chris-plus-alphanumericgibberish** [comment]
Yeah, what definitely makes this an unimplemented feature is that I've long intended to allow you to treat pets basically like a container, both for giving and taking. Currently it's left in a preliminary state where you can take any item (except gold) with #loot, equip armor (but not weapons) with #equip, and give arbitrary items by pegging them at their head with (t)hrow.

**Comment by Chris-plus-alphanumericgibberish** [comment]
Upon reflection, no this is an enhancement rather than a (now renamed) undeveloped feature, because there's no conceptual design work left to do.
